nwb_conversion_tools/datainterfaces/behavior/movie/movie_utils.py

```diff
@@ -141,3 +142,51 @@ def __exit__(self, *args):
 
     def __del__(self):
         self.vc.release()
+
+
+class MovieDataChunkIterator(GenericDataChunkIterator):
+    """DataChunkIterator specifically for use on RecordingExtractor objects."""
+
+    def __init__(
+        self,
+        movie_file: PathType,
```

Comment:
```suggestion
        movie_file: FilePathType,
```

We use the TypeVar's from `utils.json_schema` these days; you should be able to delete the `PathType` definition above.